src/metabase/api/user.clj

```diff
@@ -450,8 +450,11 @@
                                          api/*is-superuser?* (conj :login_attributes))
                               :non-nil (cond-> #{:email}
                                          api/*is-superuser?* (conj :is_superuser))))]
-          (t2/update! User id changes)
-          (events/publish-event! :event/user-update (assoc (t2/select-one User :id id) :changes changes)))
+          (let [raw-user-before-update (t2/select-one User :id id)]
```

Comment:
This is an extra db call now isn't it? Is there some way we could avoid both getting the user and updating the user? Like, is there some way to do it in one fell swoop? Is it a non-issue?

Reply:
It is, I think it might be better to use what fetch-user returns as the meaning of what a user is. Then I can use the one from line 429.